src/domains/stage/dto/index.ts

```diff
@@ -7,3 +7,21 @@ export class StageDTO {
     this.name = stage.name;
   }
 }
+
+const StageTypeEnum: {
```

Comment:
This way of representing stage types is much cleaner: `type StageType = 'BACKLOG' | 'UNSTARTED' | 'STARTED' | 'COMPLETED' | 'CANCELED' | 'OTHER'`

Reply:
fixed